pep-3156.txt

```diff
@@ -1647,15 +1720,31 @@ callback-based frameworks like Twisted.  After converting a coroutine
 into a Task, callbacks can be added to the Task.
 
 To convert a coroutine into a task, call the coroutine function and
-pass the resulting coroutine object to the ``asyncio.Task()``
-constructor.  You may also use ``asyncio.async()`` for this purpose.
+pass the resulting coroutine object to the ``loop.create_task()``
+method.  You may also use ``asyncio.ensure_future()`` for this purpose.
 
 You may ask, why not automatically convert all coroutines to Tasks?
 The ``@asyncio.coroutine`` decorator could do this.  However, this would
 slow things down considerably in the case where one coroutine calls
 another (and so on), as switching to a "bare" coroutine has much less
 overhead than switching to a Task.
 
+A Task is inherited from ``Future`` but adds new methods:
```

Comment:
In asyncio Task is technically derived from `Future` but in pep we can declare it a *future-compatibe* with explicit enumeration of required/supported methods.

I heard very many times how people are confusing by Future->Task inheritance.
It makes sense for asyncio implementation but mentally Task and Future are different flavors of async development style.

Moreover end asyncio user usually never works with futures explicitly but operates on tasks only.